File: benches/lua/prime_count.lua
local function is_prime(n)
    if n < 2 then return false end
    local i = 2
    while i * i <= n do
        if n % i == 0 then return false end
        i = i + 1
    end
    return true
end

local n = tonumber(arg[1])
local count = 0
for x = 2, n do
    if is_prime(x) then count = count + 1 end
end
print(count)

File: benches/py/prime_count.py
import sys

def is_prime(n):
    if n < 2:
        return False
    i = 2
    while i * i <= n:
        if n % i == 0:
            return False
        i += 1
    return True

n = int(sys.argv[1])
count = 0
for x in range(2, n + 1):
    if is_prime(x):
        count += 1
print(count)
